data/campaigns/Dead_Water/units/Citizen.cfg

```diff
@@ -14,7 +14,7 @@
     advances_to=Merman Brawler,Merman Fighter,Merman Hunter
     cost=9
     usage=fighter
-    description= _ "Mermen are mostly peaceful, but will fight to defend their homes. In this situation, even mermen with no weapons or training can be dangerous."
+    description= _ "Merfolk are mostly peaceful, but will fight to defend their homes. In this situation, even merfolk with no weapons or training can be dangerous."
```

Comment:
I've changed the Citizen, but left the corresponding text in the Brawler as male, it seemed to fit better to me.